### DISCUSSION TITLE
Suggestion on cmdline and search mode

### DISCUSSION THREAD

**Discussion by Nadim147c** [saghen/blink.cmp#262] [General]
Searching through docs, I couldn't find any way to register buffer completion for cmdline (:) or search (/) mode. So I'm guessing this feature doesn't exist. If I'm wrong and there is a way to get buffer completion on search, then kindly show me how. 
If there isn't any feature like that what alternative should I use?

**Comment by saghen** [comment]
You can use nvim-cmp for just the cmdline/search but there may be alternatives I don't know about. You can track that feature here #138
